das/celestia.go

```diff
@@ -38,6 +38,7 @@ type DAConfig struct {
 	NamespaceId        string           `koanf:"namespace-id" `
 	AuthToken          string           `koanf:"auth-token" reload:"hot"`
 	ReadAuthToken      string           `koanf:"read-auth-token" reload:"hot"`
+	KeyringKeyname     string     	    `koanf:"keyring-keyname" reload:"hot"`
```

Comment:
I added these in this spot of the struct so that DA-related auth stuff is near each other. Totally cool to move elsewhere if it makes more sense